Gemfile

```diff
@@ -77,6 +66,10 @@ gem 'font-awesome-sass', '5.0.9'
 gem 'font-awesome-rails'
 gem 'activerecord-session_store'
 gem 'whenever', require: false
+gem 'oauth2'
+gem 'omniauth-oauth2'
+
+gem 'omniauth-urs', :git => "https://git.earthdata.nasa.gov/scm/cmrarc/omniauth-urs.git", branch: "develop"
```

Comment:
remind me why we are going to pull from develop and not master? Should the default not pull from master and the active developer switch to develop? Or is this just till we merge develop into master at the end of the sprint and from then on it will be master by default?

Reply:
This is just temporary, I think we want to create a tag, when things are mature and working as expected.